src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -272,6 +272,124 @@ const listifyCenters = (center, idx, list) => {
   );
 };
 
+export const documentTypeEnums = [
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9',
+  '10',
+  '11',
+  '12',
+  '13',
+  '14',
+  '15',
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+  '21',
+  '22',
+  '23',
+  '24',
+  '25',
+  '26',
+  '27',
+  '28',
+  '29',
+  '30',
+  '31',
+  '32',
+  '33',
+  '34',
+  '35',
+  '36',
+  '37'
+];
+
+export const documentTypes = [
+  'Application for Automotive Adaptive Equipment',
+  'Application for Compensation for Unemployability',
+  'Application for Specially Adapted Housing',
+  'Examination for Housebound Status',
+  'Request for Nursing Home Information',
+  'Examination for Housebound Status',
+  'Temporary Total Disability for PTSD_21_0781 and PTSD_21_0781A',
+  'Service Connected PTSD Statement',
+  'Service Connected Personal Assault PTSD Statement',
```

Comment:
'Service Connected PTSD Statement' and 'Service Connected Personal Assault PTSD Statement' are not part of the EVSS attachment list, should they be included here?

Reply:
@djmunoz1 do you have any insight into this? Should we ask EVSS directly?